CircuitPython_JEplayer_mp3/code.py

```diff
@@ -317,8 +317,7 @@ def choose_folder(self, base='/sd'):
 def mount_sd():
     """Mount the SD card"""
     spi = busio.SPI(board.SCK, MOSI=board.MOSI, MISO=board.MISO)
-    sd_cs = digitalio.DigitalInOut(board.SD_CS)
-    sdcard = adafruit_sdcard.SDCard(spi, sd_cs, baudrate=6000000)
+    sdcard = sdcardio.SDCard(spi, board.SD_CS)
```

Comment:
The default `baudrate` is 8000000. I'm not sure why the original code set it to 6000000. Do you have the parts to test it?
```suggestion
    sdcard = sdcardio.SDCard(spi, board.SD_CS, baudrate=6000000)
```

tagging @jepler here

Reply:
Yeah I do have them. I'll test it now.

Reply:
@dhalbert Just tested and it works.